docs/SNAPSHOTS

```diff
@@ -0,0 +1,22 @@
+## Download latest snapshot  
+Stop IXO service  
+`systemctl stop ixo.service`  
+
+Remove old data in directory `~/.ixod/data`  
+```
+rm -rf ~/.ixod/data; \
```

Comment:
Would be great to actually add a warning to save priv_valodator_state.json

Reply:
I agree, im wondering how we can word it to justify it to validators.